plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -126,28 +127,38 @@ func (n *attributesNormalizer) normalizeTelemetryAttributes(attributes, resource
 	}
 
 	var (
-		sdkName        string
-		sdkVersion     string
+		sdkName    string
+		sdkVersion string
+		sdkLang    string
+	)
+	var (
+		sdkAutoName    string
 		sdkAutoVersion string
-		sdkLang        string
 	)
 	sdkName, sdkVersion, sdkLang = "-", "-", "-"
 	mode := instrumentationModeManual
 
-	// TODO read telemetry.auto.version from telemetry.distro.* from v1.22
```

Comment:
it's deployed now?

Reply:
We haven't started using them in ADOT SDK. Is it what you are asking?